### PR TITLE
Beta 2 (0.2) - Luna

### PR DESCRIPTION
**Pull request by amulware**
Changelog:

```
- Added sign-up
- Added creation of clients and projects
- Added warning when time entry description exceeds limit
- Added confirmation when deleting time entries
- Fixed log not updating correctly overnight
- Fixed deleted tags not syncing correctly
- Various fixes and improvements
```

### CODE REVIEW

File: Toggl.Daneel/Info.plist [hunk 1]
```diff
@@ -34,8 +34,8 @@
 	<key>CFBundleName</key>
 	<string>Toggl</string>
 	<key>CFBundleShortVersionString</key>
-	<string>0.1</string>
+	<string>0.2</string>
 	<key>CFBundleVersion</key>
-	<string>7</string>
+	<string>9</string>
```
Comment (willsb) [inline Toggl.Daneel/Info.plist]: Why skip 8?

Comment (amulware) [reply Toggl.Daneel/Info.plist]: See the different commits

Comment (amulware) [reply Toggl.Daneel/Info.plist]: We released two builds. Since we use Testflight for RC testing, each build needs a unique build number.

Comment (willsb) [reply Toggl.Daneel/Info.plist]: understandable have a nice day

**Pull request merged**